packages/dd-trace/test/opentracing/propagation/log.spec.js

```diff
@@ -53,6 +55,44 @@ describe('LogPropagator', () => {
         }
       })
     })
+
+    it('should inject 128-bit trace IDs when enabled', () => {
+      config.traceId128BitLoggingEnabled = true
+
+      const carrier = {}
+      const traceId = id('1234567812345678')
+      const traceIdTag = '8765432187654321'
+      const spanContext = new SpanContext({
+        traceId,
+        spanId: id('-456', 10)
```

Comment:
What's the reason for this `-456` thing? I'm unclear what you're trying to test here. 🤔

Reply:
That's just because it was copied from another test. I can change it if it's confusing since in this test it's not useful, but the idea of the negative value in the other test was to ensure that negative values are converted to the correct number for tracers that were sending signed integers instead of unsigned. In this test specifically it doesn't matter as this is not what is tested.

Reply:
I moved this to a separate test to avoid confusion in other tests.